src/main.cc

```diff
@@ -1443,12 +1434,55 @@ ConfigureCurrentKid(const CommandLine &cmdLine)
     }
 }
 
-static void StartUsingConfig()
+static void
+ConfigureDebugging()
+{
+    if (opt_no_daemon) {
+        /* we have to init fdstat here. */
```

Comment:
Do we know _why_ we `have to init fdstat here`? If yes, we should probably document that knowledge.